main.py

```diff
@@ -11,12 +16,12 @@ def main():
     try:
         connection_parameters = parse_connection_arguments()
     except RuntimeError as ex:
-        print(f'[Err] exception has been caught during argument parsing: {ex}')
+        logger.critical(f'Exception has been caught during argument parsing.', exc_info=True)
 
     try:
         db_connection = DBConnection(connection_parameters)
     except Exception as ex:
-        print(f'[Err] exception has been caught during creating db connection: {ex}')
```

Comment:
Remove as ex, because it is no more used

Reply:
resolved